docs/Makefile

```diff
@@ -16,7 +16,7 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 deps:
-	pip install --disable-pip-version-check -r requirements.txt
```

Comment:
just sticking to being consistent with lines 31 and 32 in .travis.yml. I can revert this if you'd prefer :)